components/camel-ahc/src/main/docs/ahc-component.adoc

```diff
@@ -3,11 +3,11 @@
 
 *Available as of Camel version 2.8*
 
-The ahc: component provides HTTP based endpoints
+The *ahc:* component provides HTTP based endpoints
 for consuming external HTTP resources (as a client to call external
```

Comment:
The *ahc:* is not rendered well on the website. 

https://camel.apache.org/staging/

Reply:
Changed to capitals.